jest/unit.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  rootDir: '../',
+  collectCoverageFrom: ['packages/**/*.js', '!src/**/*.test.js', '!src/**/*.spec.js', '!src/**/*.e2e.js'],
+  cacheDirectory: '<rootDir>/jest_cache',
+  // testRegex: '__tests__/.*\\.test\\.js$',
+
+  moduleFileExtensions: ['ts', 'js', 'node', 'json'],
+  transform: {
+    '^.+\\.(js|jsx)$': 'babel-jest',
+    '^.+\\.(ts|tsx)$': 'ts-jest',
+  },
+  testMatch: ['/**/?(*.)+(spec).[jt]s?(x)'],
+  // testEnvironment: 'node',
+  // testEnvironment: './jest/env.js',
+  moduleNameMapper: {
+    '@cennznet/api(.*)$': '<rootDir>/packages/api/src/$1',
+    '@cennznet/crml-attestation(.*)$': '<rootDir>/packages/crml-attestation/src/$1',
+    '@cennznet/crml-generic-asset(.*)$': '<rootDir>/packages/crml-generic-asset/src/$1',
```

Comment:
Maybe ignore `crml-*` runtime packages from this initially since it's not required for #70

Reply:
Similar to the above concern~